### ISSUE TITLE
Suggestion - EMS "/Me" Improvement

### ISSUE THREAD

**Issue by sleighwhite** [InZidiuZ/op-framework-issue-tracker#2627]
### Assurance

I have read the `README.md`

### Summary

An overhaul to replace EMS workers typing "/me" descriptions for their medical RP. Presented potential solutions: 

- UI for EMS to select from a predetermined list of "/Me" descriptions and coordinating animations (could look similar to how the 'H' key menu works for toggling clothing).
- Creating item assets that EMS can "use" while treating patients in the field

### Reason

As EMS, we try our best to provide quality medical role play for players across the server. Our role play is often vital in emergency situations, and we try to be descriptive while we tend to our patients both in the field and in hospitals/infirmaries. The best way to do this at the minute is to type out a "/me" for the type of action we're providing to the patient. And while this does allow us to be a bit more creative with our roleplay, for very basic and almost always necessary actions, it can be really repetitive and can take a lot of time to type or to re-enter from the /t command. 

Essentially, I think it would be make for more quality roleplay if there was a UI menu with a list of predetermined, frequently used descriptions of EMS actions. I.e., "Cleans and disinfects wounds" or "Sets up an IV." It'd be great if selecting one of these options also activated an animation as well. The animation could just be something general that already exists like "typing" or something like that. 

Another alternative would be to have actual items that are "used" by the EMS and there's an animation that goes along with each item. I don't know if it'd be more work to create assets like a "C-Collar" and "Acetaminophen" than it would be to create an entire UI menu, but I think either option would be worth exploring. Or maybe one option would be a good stop-gap solution while the other option was refined and put into early development.

The current method of EMS employees writing their own "/me" descriptions has been used for a few years now from what I have seen. I feel like an update could definitely be used. Individual EMS employees would still be able to utilize "/me" for further enhancing RP in less repeatable circumstances. Having an item or a UI to select from would just keep the flow of the RP moving along normally without awkward pauses for typing or copy/pasting. A streamlined experience in the field could help encourage other players to interact more with EMS as well. 

All of this is coming from a very new EMT, but I've been a part of the Legacy community for a couple years now. For more relevant thoughts and consideration, please reach out to EMS Command. Also, these are just the two potential solutions that came to my mind when I was considering the issue. There are probably much more sophisticated ways of streamlining the process.

**Comment by roraraptor** [comment]
If any emotes created are also available for use by those of us who use Stream Decks and similar tools for their /me descriptions, then I am also very for this.

**Comment by coalaura** [comment]
Thats what keybinds are for tho, no? If you have something you do very often you can add a keybind or if you use a streamdeck you can use the integration i built a while back https://github.com/coalaura/opfw_streamdeck

**Comment by sleighwhite** [comment]
Yeah! Keybinds can be a stop gap for sure, but (and I could be mistaken) those binds are cross-character. Which means if I have like 8 different keybinds for my EMS character, I can't really afford to have too many more for any other character to use. And I stand the risk of accidentally "inserting an IV" on someone on my crim if I fat finger something haha. 

I just thought it might be nice to have something more integrated for EMS.
